gradle/verification-metadata.xml

```diff
@@ -60,6 +60,7 @@
          <ignored-key id="5dfb46c505b694fe" reason="Key couldn't be downloaded from any key server"/>
          <ignored-key id="36d4e9618f3adab5" reason="Key couldn't be downloaded from any key server"/>
          <ignored-key id="02216ed811210daa" reason="Key couldn't be downloaded from any key server"/>
+         <ignored-key id="1a2a1c94bde89688" reason="Key couldn't be downloaded from any key server"/>
```

Comment:
Since we don't use key servers, how do we assure the local artifact is legit?

Reply:
We use the hash of the artifact.  Ideally, the hash of the artifact comes from someplace we trust, so if the artifact we resolve has the same hash as in this file, we have a legitimate/trustworthy artifact, even if we can't verify it by the signing key.

Reply:
Verification metadata updates are no longer needed, since we are not updating artifact versions